queries.sql

```diff
@@ -15,3 +15,34 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 -- Query 8: Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+/*Vet clinic database: query and update animals table*/
+-- Query 1: update the animals table by setting the species column to unspecified
+UPDATE animals SET species = 'unspecified';
+-- Query 2: Update the animals table by setting the species column to digimon for all animals that have a name ending in mon
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+-- Query 3: Update the animals table by setting the species column to pokemon for  all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species='';
+-- Query 4:  Inside a transaction delete all records in the animals table then rollback
+BEGIN TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Take%20a%20screenshot%20of%20the,the%20results%20of%20your%20actions.) 

> you are to delete all records in the animals table, then roll back the transaction.
> After the rollback verify if all records in the animals table still exists. below is an example 👇  
```
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```